app/views/deploys/new.html.erb

```diff
@@ -18,6 +18,8 @@
         </div>
       <% end %>
 
+      <%= Samson::Hooks.render_views(:before_deploy_form, self, project: @project, form: form) %>
```

Comment:
I'd think only after (what could be more important then the revision?) + rename to deploy_form